docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md

```diff
@@ -57,6 +59,18 @@ Parameter                     | Type                 | Description
 
 If `attrName` or `attrValue` is a String, `withAttribute` selects an element by strict match.
 
+#### filterVisible
+
+Method                              | Type     | Description
+----------------------------------- | -------- | -----------
+`filterVisible()` | Selector | Creates a selector that filters a matching set leaving only visible elements.
```

Comment:
Please explain the term `visible` here. At first sight I'd think it `visibility:visible` and `visibility:hidden` correspondingly.